Source/UIImage+ASConvenience.mm

```diff
@@ -114,9 +115,12 @@ + (UIImage *)as_resizableRoundedImageWithCornerRadius:(CGFloat)cornerRadius
     // UIBezierPath objects are fairly small and these are equally sized. 20 should be plenty for many different parameters.
     __pathCache.countLimit = 20;
   });
-  
+
   // Treat clear background color as no background color
-  if ([cornerColor isEqual:[UIColor clearColor]]) {
+  // clearColor is a singleton UIColor therefore we can replace the isEqual: call with pointer comparison as
+  // [UIColor isEqual:] does a bunch of things like checking isKindOfClass:, reading out the cachedcolor and
+  // ultimately just calls CGColorEqualToColor().
+  if (cornerColor == UIColor.clearColor) {
```

Comment:
Maybe we should even use `CGColorGetAlphaComponent == 0`

Reply:
This would be a more safe check, agree.